nitrocli/nitrocli.1

```diff
@@ -0,0 +1,30 @@
+.TH NITROCLI 1
+.SH NAME
+nitrocli \- access Nitrokey devices
+.SH SYNOPSIS
+.B nitrocli
+\fBclear\fR|\fBclose\fR|\fBopen\fR|\fBstatus\fR
```

Comment:
It appears uncommon to have the subcommand appear bold. Any particular reason to do that? Or am I just looking at the wrong role model?

Reply:
`man-pages(7)` says:
> **SYNOPSIS**
> […]
> For  commands, this shows the syntax of the command and its arguments (including options);
> boldface is used for as-is text and italics are used to indicate replaceable arguments.

As the commands are “as-is text”, I chose boldface.